### DISCUSSION TITLE
Add a new value to a column of type Enum

### DISCUSSION THREAD

**Discussion by WaleedAlrashed** [laravel/framework#44142] [Q&A]
Hi Everyone

**Problem:** I am looking for a  `Blueprint` method to add a new variable to a column of type `enum` 

**What I found:**

After doing some research, I have found the following:

I can use `DB` statements like the [this](https://stackoverflow.com/questions/54806693/alter-enum-column-and-add-value-to-that-column-in-laravel ) to alter the required column , But I don't suppose that there is a `Schema` method.

Is there any way to do this with Schema?

**Comment by shaedrich** [comment]
There kind of is: "Creating" the column anew by calling `$table->enum()` again with the currently available list of cases, including the new one.

This, however, requires knowing the list of cases until know before adding the new one and is everything but DRY. So, yeah: There's no clean solution via `Blueprint`
